src/utils/AudioCompare.js

```diff
@@ -145,10 +145,10 @@ export default class AudioRecorderCompair extends Component {
           audioSrc: temp_audioSrc,
         });
 
-        if (!this.props.isAudioPreprocessing) {
```

Comment:
comment entire block and use only ` this.props.setRecordedAudio(temp_audioSrc);`